stochastic_physics/stochastic_physics_wrapper.F90

```diff
@@ -89,20 +90,25 @@ subroutine stochastic_physics_wrapper (GFS_Control, GFS_Data, Atm_block, ierr)
 #endif
     ierr = 0
 
+    levs   = GFS_Control%levs
```

Comment:
I understand it's convenient and only little memory overhead, but why declare redundant local variables? It doesn't give any better computational performance.

Reply:
Well, it just makes the code more readable and less clutter.